tools/template_convert.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python3
+
```

Comment:
This should come with a few lines of documentation in the source code. 2019, no one will remember the struggle. And I want one place people will find when they grep for gengen.